src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -115,3 +123,37 @@ describe('DownloadCertificate', () => {
     });
   });
 });
+
+describe('UpdateSelectedTestResultModel', () => {
+  const testTreeNode = {
+    key: '1',
+    value: 'test'
+  } as KeyValue<string, string>;
+
+  const newTestData = {
+    treeData: testTreeNode,
+    testRecord: {
+      testRecord: TEST_MODEL_UTILS.mockTestRecord(),
+      testType: TEST_MODEL_UTILS.mockTestType()
+    }
+  } as TestTypeChange;
+  it('action should have correct type & payload', () => {
```

Comment:
similar concept here and in this file `it should create UpdateSelectedTestResultModel action`